packages/main/src/plugin/index.ts

```diff
@@ -1289,7 +1289,7 @@ export class PluginSystem {
       ): Promise<unknown> => {
         // create task
         const task = taskManager.createTask({
-          title: `Building ${imageName}`,
+          title: `Building ${imageName ?? 'unnamed image'}`,
```

Comment:
IMO: `Building image` better that `Building unnamed image`
```suggestion
          title: `Building image ${imageName ?? ''}`,
```